mne/simulation/evoked.py

```diff
@@ -34,9 +36,8 @@ def simulate_evoked(fwd, stc, info, cov, snr=3., tmin=None, tmax=None,
         Measurement info to generate the evoked.
     cov : Covariance object
         The noise covariance.
-    snr : float
-        signal to noise ratio in dB. It corresponds to
-        10 * log10( var(signal) / var(noise) ).
+    nave : int
+        Number of averaged epochs.
```

Comment:
Just wanted to add that [we're good here now](https://4638-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/generated/mne.simulation.simulate_evoked.html#mne.simulation.simulate_evoked).